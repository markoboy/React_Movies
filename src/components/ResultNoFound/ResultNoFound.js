import React, { memo } from 'react';

function ResultNoFound() {
  return (
    <div className="flex flex--grow justify-content--center align-items--center">
      <h2 className="font--regular">No Movie Found</h2>
    </div>
  );
}

export default memo(ResultNoFound);
