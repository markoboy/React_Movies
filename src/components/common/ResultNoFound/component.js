import React, { memo } from 'react';

function ResultNoFoundComponent() {
  return (
    <div className="flex flex--grow justify-content--center align-items--center">
      <h2 className="font--regular">No Movie Found</h2>
    </div>
  );
}

export default memo(ResultNoFoundComponent);
