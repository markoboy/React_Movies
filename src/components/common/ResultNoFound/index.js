export { default } from './ResultNoFound.component';
