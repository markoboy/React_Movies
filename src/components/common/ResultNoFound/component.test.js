import React from 'react';
import { render } from '@testing-library/react';

import ResultNoFound from './component';

describe('ResultNoFound Component', () => {
  it('renders with No Movie Found text', () => {
    const { container, getByText } = render(<ResultNoFound />);

    const textContainer = getByText(/No Movie Found/);

    expect(textContainer).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
