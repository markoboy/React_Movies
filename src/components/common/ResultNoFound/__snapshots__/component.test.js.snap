// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultNoFound Component renders with No Movie Found text 1`] = `
<div>
  <div
    class="flex flex--grow justify-content--center align-items--center"
  >
    <h2
      class="font--regular"
    >
      No Movie Found
    </h2>
  </div>
</div>
`;
